pkg/moov/disputes.go

```diff
@@ -185,7 +185,44 @@ func (c Client) AcceptDispute(ctx context.Context, disputeID string) (*Dispute,
 // UploadDisputeEvidence Uploads text as evidence for a dispute.
 // https://docs.moov.io/api/money-movement/disputes/post-text/
 func (c Client) UploadDisputeEvidence(ctx context.Context, disputeID string, evidenceText DisputesEvidenceText) ([]DisputeEvidence, error) {
-	resp, err := c.CallHttp(ctx, Endpoint(http.MethodPost, pathDisputeUploadEvidenceText, disputeID), AcceptJson(), JsonBody(evidenceText))
+	resp, err := c.CallHttp(ctx, Endpoint(http.MethodPost, pathDisputeEvidenceText, disputeID), AcceptJson(), JsonBody(evidenceText))
+
+	if err != nil {
+		return nil, err
+	}
+
+	return CompletedListOrError[DisputeEvidence](resp)
+}
+
+// DeleteDisputeEvidence deletes a piece of dispute evidence for the given dispute and evidence id
+// https://docs.moov.io/api/money-movement/disputes/delete
+func (c Client) DeleteDisputeEvidence(ctx context.Context, disputeID, evidenceID string) error {
+	resp, err := c.CallHttp(ctx, Endpoint(http.MethodDelete, pathDisputeEvidence, disputeID, evidenceID), AcceptJson())
+	if err != nil {
+		return err
+	}
+	return CompletedNilOrError(resp)
+}
+
+// UploadEvidenceFile uploads a new evidence file for the given dispute id
+// https://docs.moov.io/api/money-movement/disputes/post-file/
+func (c Client) UploadEvidenceFile(ctx context.Context, disputeID string, evidenceType EvidenceType, filename string, file []byte) error {
+	var multiParts []multipartFn
+	multiParts = append(multiParts, MultipartField("evidenceType", string(evidenceType)))
+	multiParts = append(multiParts, MultipartFile("file", filename, bytes.NewReader(file)))
+
```

Comment:
Is there a reason `file []byte` couldn't be `file io.Reader` so applications can more efficiently stream off disk / network?